virttest/qemu_monitor.py

```diff
@@ -177,6 +177,49 @@ def wait_for_create_monitor(vm, monitor_name, monitor_params, timeout):
         raise MonitorConnectError(monitor_name)
 
 
+def x_non_x_feature(feature):
+    """
+    Reports the other of the x-/non-x- prefixed feature to the given feature
+
+    :param feature: asked-for feature
+    :return: when $feature startswith x- it reports non-x- variant, otherwise
+             it prefixes x-
+    """
+    if feature.startswith("x-"):
+        return feature[2:]
+    else:
+        return "x-%s" % feature
+
+
+def pick_supported_x_feature(feature, supported_features,
+                             disable_auto_x_evaluation,
+                             error_on_missing=False, feature_type="Feature"):
+    """
+    Attempts to chose supported feature with/without "x-" prefix based
+    on list of supported features.
+
+    :param feature: feature that allows x- or non-x- prefix
+    :param supported_features: list of supported features
+    :param error_on_missing: whether to fail when no variant is supported
```

Comment:
docstring for `disable_auto_x_evaluation` would help

Reply:
oups, adding